packages/dx-react-chart/src/plugins/axis.jsx

```diff
@@ -69,10 +74,10 @@ export class Axis extends React.Component {
       name,
       indentFromAxis,
       isArgumentAxis,
-      rootComponent: Root,
-      tickComponent: Tick,
-      labelComponent: Label,
-      lineComponent: Line,
+      rootComponent: RootComponent,
```

Comment:
Why do you use *Component in names?

Reply:
`Root` is occupied in the outer scope (`import` section).